python/cuml/test/test_stratified_kfold.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cudf
+import cupy as cp
+import pytest
+
+from cuml.model_selection import StratifiedKFold
+
+
+@pytest.mark.parametrize("shuffle", [True, False])
+@pytest.mark.parametrize("n_splits", [3, 5, 10])
```

Comment:
Why this values? Seems like 3, 5, and 10 are testing essentially the same thing.

Would be good to have tests of invalid input, say 0 splits for example.

Reply:
Because these values are the most common choices in kaggle competitions. Yes, I agree they mean the same thing. I'll add a separate test for the invalid number of folds.